pulp/src/x86.rs

```diff
@@ -136,6 +136,7 @@ impl Default for Arch {
 include!(concat!(env!("OUT_DIR"), "/x86_64_asm.rs"));
 
 #[cfg(test)]
+#[cfg_attr(not(feature = "complex"), allow(unused_imports))]
```

Comment:
I'm being somewhat lazy here but without the complex stuff there were a lot of imports that became unused (in this test module in particular).